packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2631,7 +2631,9 @@ class RenderSemanticsGestureHandler extends RenderProxyBox implements SemanticsA
        _onLongPress = onLongPress,
        _onHorizontalDragUpdate = onHorizontalDragUpdate,
        _onVerticalDragUpdate = onVerticalDragUpdate,
-       super(child);
+       super(child) {
+    assert(scrollFactor != null);
```

Comment:
I'm trying to imagine why my brain did this one the old way and am totally at a loss. Will do.